plugins/chat/assets/javascripts/discourse/lib/init-sidebar-state.js

```diff
@@ -1,7 +1,14 @@
+import { ADMIN_PANEL, MAIN_PANEL } from "discourse/services/sidebar-state";
 import { getUserChatSeparateSidebarMode } from "discourse/plugins/chat/discourse/lib/get-user-chat-separate-sidebar-mode";
 
+export const CHAT_PANEL = "chat";
+
 export function initSidebarState(api, user) {
-  api.setSidebarPanel("main");
+  if (api.getSidebarPanel()?.key === ADMIN_PANEL) {
```

Comment:
Since this is called as soon as chat is initialized, and the admin one is called first, we want to make sure we don't override the admin one. There may need to be a more refined way of doing this...